### PR TITLE
docs: updated docs to include enabled feat

### PR DESCRIPTION
**Pull request by MichaelUnkey**
eng-392

## What does this PR do?

updated docs to reflect enabled feat on keys

Fixes # (issue)

*If there is not an issue for this, please create one first. This is used to tracking purposes and also helps use understand why this PR exists*

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Chore (refactoring code, technical debt, workflow improvements)
- [ ] Enhancement (small improvements)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How should this be tested?

Make sure docs look correct for the feature

### Required

- [x] Filled out the "How to test" section in this PR
- [x] Read [Contributing Guide](./CONTRIBUTING.md)
- [x] Self-reviewed my own code
- [x] Commented on my code in hard-to-understand areas
- [x] Ran `pnpm build`
- [x] Ran `pnpm fmt`
- [x] Checked for warnings, there are none
- [x] Removed all `console.logs`
- [x] Merged the latest changes from main onto my branch with `git pull origin main`
- [x] My changes don't cause any responsiveness issues

### Appreciated

- [ ] If a UI change was made: Added a screen recording or screenshots to this PR
- [ ] Updated the Unkey Docs if changes were necessary

### CODE REVIEW

File: apps/docs/mint.json [hunk 1]
```diff
@@ -141,10 +141,8 @@
               "pages": [
                 "libraries/ts/sdk/keys/create",
                 "libraries/ts/sdk/keys/verify",
-                "libraries/ts/sdk/keys/get",
-                "libraries/ts/sdk/keys/verifications",
                 "libraries/ts/sdk/keys/update",
-                "libraries/ts/sdk/keys/delete"
+                "libraries/ts/sdk/keys/revoke"
```
Comment (perkinsjr) [inline apps/docs/mint.json:145]: This should be `libraries/ts/sdk/keys/delete`

**Review by perkinsjr — CHANGES_REQUESTED** [review]

File: apps/docs/features/enabled.mdx [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+  "error": {
+    "code": "DISABLED",
+    "docs": "https://unkey.dev/docs/api-reference/errors/code/DISABLED",
+    "message": "<string>",
+    "requestId": "req_1234"
+  }
+}
+```
+
+<Note>
```
Comment (chronark) [inline apps/docs/features/enabled.mdx:49]: this seems left over from copy paste?

**Review by chronark — CHANGES_REQUESTED** [review]

File: apps/docs/features/enabled.mdx (hunk 2 above)
Comment (MichaelUnkey) [reply apps/docs/features/enabled.mdx:49]: Sure is. Had a meeting with James and it got forgotten :)

File: apps/docs/mint.json (hunk 1 above)
Comment (MichaelUnkey) [reply apps/docs/mint.json]: changed

**Pull request merged**